### PR TITLE
Add cspi directories

### PR DESCRIPTION
**Pull request by rnetser**
The CPSI team is starting to add tests using openshift ci framework.  
The team is running other team's tests and thus is not tied to a GitHub org.  
Adding the directories planned structure and skipping GitHub integration for periodic jobs (currently no presubmit tests are planned)

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -2536,6 +2540,7 @@ periodics:
       - --ignore=containers/gvisor-tap-vsock
       - --ignore=containers/podman-py
       - --ignore=red-hat-storage/mcg-osd-deployer
+      - --ignore==cspi
```
Comment (jmguzik) [inline ci-operator/jobs/infra-periodics.yaml:2543]: I think we need only one "=" here

**Review by jmguzik — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/infra-periodics.yaml [hunk 2]
```diff
@@ -1241,6 +1241,10 @@ periodics:
       - --ignore-repo=ci-operator/jobs/openshift/cluster-api-provider-kubevirt
       - --ignore-org=openshift-priv
       - --ignore-org=kubernetes-sigs
+      - --ignore-org=ci-operator/config/layered-products
```
Comment (jmguzik) [inline ci-operator/jobs/infra-periodics.yaml:1244]: you need layered-products or cspi only? I do not see "layered-products" in the PR, it's for the future?

**Comment by smg247** [comment]
You also need to add the ignore for the presubmit to: https://github.com/openshift/release/blob/244b981d641870862be5970c7587f6551c9111d2/ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml#L26-L29

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (rnetser) [reply ci-operator/jobs/infra-periodics.yaml]: fixed

File: ci-operator/jobs/infra-periodics.yaml (hunk 2 above)
Comment (rnetser) [reply ci-operator/jobs/infra-periodics.yaml]: leftover, removed

**Comment by rnetser** [comment]
>

**Review by jmguzik — APPROVED** [review]

**Pull request merged**